### ISSUE TITLE
Error when clicking Connect in Dax Studio

### ISSUE THREAD

**Issue by wildrose1** [DaxStudio/DaxStudio#245]
I have Excel 2010 and Dax Studio 2.9.4. When I click Connect I get the following error:

11/12/2019 12:08:20 PM		Method not found: 'System.Threading.Tasks.Task`1<!!0> System.Net.Http.HttpContentExtensions.ReadAsAsync(System.Net.Http.HttpContent)'.

However if I open the program outside Excel I can connect to PowerBI, but I don't get Excel as an option for Output.

**Comment by dgosbell** [comment]
Other people have reported this issue here #229 but I am unable to reproduce this on any machine that I have access to, they all work perfectly.

If you are just after an export that can be read by Excel the csv export should work well for this.

Apparently rolling back to v2.9.2 worked for some of these people 

I've suggested trying a full uninstall/re-install but I don't know if anyone has tried that.

I've tried rolling back the Microsoft library we are using for the "ReadAsAsync" call to the version we were using in 2.9.2 so the new preview build [here](https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/28649169/artifacts) might fix this.

It could possibly be related to an issue with the installer. I've asked on the other thread for someone to zip up their ```c:\program files\Dax Studio``` folder and upload it to this issue so I can check, but I've had no response so far.

**Comment by wildrose1** [comment]
Attached is the zipped folder of 2.9.4. I did notice when running DAX Checker there is a "FAIL" see screenshot. 

I installed 2.9.2  and the Connect window came up but was getting errors. 

I tried an even older version 2.9.1 and was able to connect but still get and error message in the output screen. see screenshot. 

The 2.9.5 version was blocked by Windows Defender so I used caution and didn't install.

Hope all this helps.

![2019-11-13 07_53_29-DaxStudio - 2 9 1](https://user-images.githubusercontent.com/57682041/68770050-0f473600-05eb-11ea-8bce-e663f3856785.png)

[DAX Studio.zip](https://github.com/DaxStudio/DaxStudio/files/3841437/DAX.Studio.zip)
![2019-11-13 07_47_38-DaxStudio - 2 9 2](https://user-images.githubusercontent.com/57682041/68769778-89c38600-05ea-11ea-8e6a-d38744b752ae.png)
![2019-11-13 07_39_09-DAX Studio Checker](https://user-images.githubusercontent.com/57682041/68769780-89c38600-05ea-11ea-9343-37d07225b585.png)

**Comment by wildrose1** [comment]
Notice on the Fail message it is looking for a bin folder. There isn't one.

**Comment by dgosbell** [comment]
> Notice on the Fail message it is looking for a bin folder. There isn't one.

Yes, the checker application scans multiple paths. As long as there is one PASS everything should work fine.

Can you use the [following process](https://daxstudio.org/documentation/troubleshooting/logging-settings/) to enable debug logging. The reproduce this issue, then shut down DAX Studio and attach the 2 log files to this issue (there should be a DaxStudio-<yyyymmdd> and a DaxStudioExcel-<yyyymmdd> log file created if you start the logging while Excel is loading)

**Comment by wildrose1** [comment]
Here you go....
[DaxStudioExcel-20191118.log](https://github.com/DaxStudio/DaxStudio/files/3858785/DaxStudioExcel-20191118.log)
[DaxStudioExcel-20191118_001.log](https://github.com/DaxStudio/DaxStudio/files/3858786/DaxStudioExcel-20191118_001.log)

**Comment by dgosbell** [comment]
I think I see the issue. In your ```c:\program files\DAX Studio``` folder there is a file called ```DaxStudio.dll.xl2010.config``` if you rename this to ```DaxStudio.dll.config``` then shutdown and restart Excel I think it will fix this issue. (this is probably a bug in the installer)

**Comment by wildrose1** [comment]
I tried the suggested but unfortunately that didn't work. I did also restart my computer to no avail.

**Comment by dgosbell** [comment]
I've tried rolling back the Microsoft library we are using for ReadAsAsync in the 2.10 release which I just published so please give that a try and see if it has any effect on this issue

**Comment by dgosbell** [comment]
Because I cannot reproduce this issue I do not know if the attempted fix in the 2.10.x releases has had any impact on this issue. So due to the lack of any further feedback I'm going to close this issue. If you are still having issues with the 2.10.2 release or later feel free to open a new issue.
